components/ws-manager-mk2/service/manager.go

```diff
@@ -241,7 +241,7 @@ func (wsm *WorkspaceManagerServer) StartWorkspace(ctx context.Context, req *wsma
 	}
 
 	var wsr workspacev1.Workspace
-	err = wait.PollWithContext(ctx, 100*time.Millisecond, 5*time.Second, func(c context.Context) (done bool, err error) {
+	err = wait.PollWithContext(ctx, 200*time.Millisecond, 15*time.Second, func(c context.Context) (done bool, err error) {
```

Comment:
increasing the poll timeout, occasionally it takes longer than 5 seconds for a workspace to get its URL field set

Reply:
@WVerlaek what is the side effect if you leave the poll at 100, instead of 200?

Reply:
hmm actually it shouldn't be much, the `client.Get()` should read from memory (cache), so can increase the poll rate to 100ms 👍 

(my worry was polling too often without e.g. exponential backoff, after increasing the timeout to 15 seconds)